libs/utils/android/workloads/youtube.py

```diff
@@ -41,56 +39,53 @@ class YouTube(Workload):
 
     def __init__(self, test_env):
         super(YouTube, self).__init__(test_env)
-        logging.debug('%14s - Workload created', 'YouTube')
+        self.logger.debug('%14s - Workload created', 'YouTube')
 
     def run(self, exp_dir, video_url, video_duration_s, collect=''):
 
         # Initialize energy meter results
         nrg_report = None
 
         # Unlock device screen (assume no password required)
-        self.target.execute('input keyevent 82')
+        System.menu(self.target)
         # Press Back button to be sure we run the video from the start
-        self.target.execute('input keyevent KEYCODE_BACK')
+        System.back(self.target)
+
+        # Use the monkey tool to start YouTube without playing any video.
```

Comment:
I would keep this chunk, where you use monkey, on a separate patch to better describe it's purpose and need.